RU: add links to redis university courses/learning paths (#1093)
redis/docs: content/develop/get-started/rag.md

- [RAG with Redis and SpringAI](https://redis.io/blog/building-a-rag-application-with-redis-and-spring-ai/)
- [Build a multimodal RAG app with LangChain and Redis](https://redis.io/blog/explore-the-new-multimodal-rag-template-from-langchain-and-redis/)
- [Get hands-on with advanced Redis AI Recipes](https://github.com/redis-developer/redis-ai-resources)

## Continue learning with Redis University

> See the [Vector Advanced Topics course](https://university.redis.io/course/i3fv2hbhqnpni8) course for modules.

Reviewer: ```suggestion
See the [Vector Advanced Topics course](https://university.redis.io/course/i3fv2hbhqnpni8) to learn more.
```
